notebooks/information_extraction_raven.ipynb

```diff
@@ -0,0 +1,2726 @@
+{
```

Comment:
<p>Can you briefly explain here what you're trying to achieve with this prompt? Where to place "START", "TEXT" tags and why we need them etc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
This was an empirical effort to improve the prompt for my application.